.gitignore

```diff
@@ -10,3 +10,4 @@ build
 dev-app-update.yml
 resource/certificates/mac.p12
 resource/certificates/win.p12
+test/logs/
```

Comment:
don't know if it matters or not but the trailing slash I'm pretty sure isn't necessary and may be unwanted, differentiating things _in_ `test/logs` vs `test/logs` itself

Reply:
Using `test/logs` might be sufficient, however in this case was attempting to be consistent with the rest of the file: `node_modules/`, `publish/node_modules/`. 

If the trailing slash doesn't make a difference, I'd lean toward being stylistically consistent.

Reply:
eh, looks like we're inconsistent in this file already, but `node_modules/` and `publish/node_modules/` are the outliers. use whichever; though the slash is significant

> If there is a separator at the end of the pattern then the pattern will only match directories, otherwise the pattern can match both files and directories.
https://git-scm.com/docs/gitignore

I think the `publish/node_modules/` is even obsoleted by `node_modules/` 🤔